moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -598,6 +596,12 @@ void ServoCalcs::applyVelocityScaling(Eigen::ArrayXd& delta_theta, double singul
   }
 
   delta_theta = collision_scale * singularity_scale * delta_theta;
+
+  if (status_ == StatusCode::HALT_FOR_COLLISION)
+  {
+    ROS_ERROR_STREAM_THROTTLE_NAMED(5, LOGNAME, "Halting for collision!");
```

Comment:
nit: 5 feels really really long. Like long enough that the user might not see this message before assuming that their code must be buggy. Maybe 2 or 3 is a more sensible period.

Reply:
Also, this isn't really an error because halting for potential collisions is an expected behavior. 
A good shorthand is:
 - FATAL for when the code should stop executing because something terrible has happened
 - ERROR for things that are outside normal operating conditions but the error is recoverable enough that code can keep executing.
 - WARNING for when something bad is happening that should be avoided but it is within normal operating conditions. 
 - INFO for output that might be important to a casually observer looking at the console output to understand what is going on with the code
 - DEBUG for output that might be interesting to a developer who is debugging this section of code

Reply:
OK, I'll make it 3 seconds. And roger that, I'll turn it into a WARNING. Thanks